examples/dht_node.rs

```diff
@@ -81,18 +75,20 @@ fn main() {
     // create PK from bytes
     let bootstrap_pk = PublicKey::from_slice(&bootstrap_pk_bytes).unwrap();
 
-                 //"51.15.37.145:33445".parse().unwrap()
     let saddr: SocketAddr = "198.98.51.198:33445".parse().unwrap();
     let bootstrap_pn = PackedNode::new(true, saddr, &bootstrap_pk);
-    assert!(server_obj.borrow_mut().kbucket.try_add(&bootstrap_pn));
+    {
```

Comment:
You should create a function named like: `try_add_to_kbucket`. You may chose another name if you want.